src/probot.ts

```diff
@@ -52,7 +50,7 @@ export class Probot {
     let level = options.logLevel;
     const logMessageKey = options.logMessageKey;
 
-    this.log = aliasLog(options.log || getLog({ level, logMessageKey }));
+    this.log = options.log || getLog({ level, logMessageKey });
```

Comment:
should be investigated if this is proper and what does aliasLog do.

Reply:
`aliasLog()` was used to have the `app.log()` function equivalent to `app.log.info()`

Your change is correct